gpt4all-backend/llama.cpp.cmake

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.12) # Don't bump this version for no reason
+cmake_minimum_required(VERSION 3.14)  # for add_link_options and implicit target directories.
 
```

Comment:
Thanks for letting me know why you are bumping, but this comment should remain as-is.

Reply:
I just copied the line from upstream. Changes here:
https://github.com/ggerganov/llama.cpp/pull/3444
https://github.com/ggerganov/llama.cpp/pull/3419